File: lab 4.java
import java.util.Arrays;
import java.util.Collections;

public class SortArray {

    public static void sortAscending(Integer[] arr) {
        Arrays.sort(arr);
    }

    public static void sortDescending(Integer[] arr) {
        Arrays.sort(arr, Collections.reverseOrder());
    }

    public static void printArray(Integer[] arr) {
        for (int i : arr) {
            System.out.print(i + " ");
        }
        System.out.println();
    }

    public static void main(String[] args) {
        Integer[] arr = {5, 3, 8, 1, 2};

        System.out.println("Original array: ");
        printArray(arr);

        sortAscending(arr);
        System.out.println("Array in ascending order: ");
        printArray(arr);

        sortDescending(arr);
        System.out.println("Array in descending order: ");
        printArray(arr);
    }
}

File: lab 4.cpp
#include <iostream>
#include <algorithm>
using namespace std;

void sortAscending(int arr[], int n) {
    sort(arr, arr + n);
}

void sortDescending(int arr[], int n) {
    sort(arr, arr + n, greater<int>());
}

void printArray(int arr[], int n) {
    for (int i = 0; i < n; i++)
        cout << arr[i] << " ";
    cout << endl;
}

int main() {
    int arr[] = {5, 3, 8, 1, 2};
    int n = sizeof(arr) / sizeof(arr[0]);

    cout << "Original array: ";
    printArray(arr, n);

    sortAscending(arr, n);
    cout << "Array in ascending order: ";
    printArray(arr, n);

    sortDescending(arr, n);
    cout << "Array in descending order: ";
    printArray(arr, n);

    return 0;
}
